### PR TITLE
Make build script platform independent

### PR DESCRIPTION
**Pull request by qwrtln**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env bash
 
+unameOut="$(uname -s)"
+case "${unameOut}" in
+    Linux*)     command=xdg-open;;
+    Darwin*)    command=open;;
+esac
+echo ${command}
+
 LANGUAGE=$1
```
Comment (evilmint) [inline build.sh:10]: exit 1 if arg wasn't found? else the process carries on

File: build.sh [hunk 2]
```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env bash
 
+unameOut="$(uname -s)"
+case "${unameOut}" in
+    Linux*)     command=xdg-open;;
+    Darwin*)    command=open;;
+esac
+echo ${command}
```
Comment (evilmint) [inline build.sh:8]: I guess was needed for debug only

**Review by evilmint — APPROVED** [review]
works!

**Comment by DarthGandalf** [comment]
I'm fine with having this script here, but it's useless to me due to `rm` part of it. Every rerun of the script is very slow, and makes iteration on the text slow. Why is rm needed?

If you want me to use it, remove rm; or I can just stick to using latexmk directly.

File: build.sh (hunk 1 above)
Comment (qwrtln) [reply build.sh:10]: It's fine. Fails exactly where it should.

**Comment by qwrtln** [comment]
> I'm fine with having this script here, but it's useless to me due to `rm` part of it. Every rerun of the script is very slow, and makes iteration on the text slow. Why is rm needed?
> 
> If you want me to use it, remove rm; or I can just stick to using latexmk directly.

If all you need is `latexmk`, you don't need a script anyway.

**Pull request merged**